packages/web/graphql/language/languages.graphql

```diff
@@ -1,4 +1,4 @@
-query languages($hasPosts: Boolean, $authoredOnly: Boolean) {
+query languages($hasPosts: Boolean, $authoredOnly: Boolean, $topics: [Int!]) {
```

Comment:
Restoring this line was necessary to generate the Apollo hooks at this stage. I remember we removed it when seeing that it didn't appear to be doing anything in the languages query, but it appears to be used in the ObjectType in a way that I don't fully understand, but I think that's why it was here originally.

Reply:
I guess the `postCount` property on languages uses it? But I don't know what that's for. I think we should try to get rid of this if it's not doing anything useful